### PR TITLE
New fonts and styles

### PR DESCRIPTION
**Pull request by limzykenneth**
Touching base. Layout not final yet. Next push will be sometime away while I work on some graphics for the landing page.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -43,8 +43,6 @@
     t.boolean  "fogged"
   end
 
-  add_index "checkins", ["device_id"], name: "index_checkins_on_device_id", using: :btree
```
Comment (t-harps) [inline db/schema.rb]: Any reason these are being deleted?

Comment (t-harps) [inline db/schema.rb]: Not sure how these got deleted but you probably need to run rake:db migrate unless there is some reasoning behind it.

Comment (limzykenneth) [inline db/schema.rb]: Don't remember changing that, I'll try rake db:migrate and see if it goes away but I remember migrating before pushing the branch though

Comment (limzykenneth) [inline db/schema.rb]: just ran `rake db:migrate` and nothing is changed, I went digging and found the commit to come from the dev branch i merged before pushing here.

Comment (t-harps) [inline db/schema.rb]: @limzykenneth not sure why it's not working as the migration file for redirect_url is in your branch but shouldn't be an issue as long as we migrate dev and the schema isn't changed before it goes into master. Will merge this request now.

**Comment by GarySiu** [comment]
Since you're still busy with the redesign, I've decided to do some of the grunt work on converting bootstrap to materialize on some of the other pages. I'll try and keep to materialize norms where possible: https://github.com/earlymarket/CoPosition/commit/c09d6da85ac9da577957f9b6a7129ebfc3746e95

I'm also trying to integrate things like your colour choices in the android prototype and would appreciate your feedback: https://invis.io/3J5L091H9

**Comment by limzykenneth** [comment]
@GarySiu 
Yup, you can go ahead and convert the other pages to use materialize classes etc. I'll come back to style them after finishing the landing page. Thanks.

I had a quick look at the android prototype and here's a few points: 

1. The text and icon in buttons might need to be black to be legible enough for visually impaired people, I'll do some test before coming back to you on this.
2. For the forms I think we should use the primary colour for the focused underlining, accent colour for when an invalid input is detected.
3. The use of accent colour should be as little as possible, only to be used for catching attention because it is chosen to be very eye catching over the primary colour and background colour. Avoid using them for labels, you can use different shades of the primary colour instead.
4. Might be helpful to group tickboxes and dropdown list together if it made sense, ie 

```
dropdown
dropdown
dropdown
tickbox
tickbox
```

**Comment by GarySiu** [comment]
@limzykenneth 

If you could comment directly on the prototype it would be really helpful. If you go to InVision in the bottom right there's a toggle for putting it in 'comment mode' and then we get a nice itemised list of things to work through. That way we don't miss any bit of input going back and forth and we track which things get addressed over time.

Where possible I've tried to stick to Google's Material Design guidelines and examples: https://www.google.com/design/spec/material-design/introduction.html

But rules are made to be broken! If are good reasons to change things, for the sake of accessibility for example, I'm more than happy to remove some neon orange :+1:

**Comment by limzykenneth** [comment]
:+1:

**Comment by GarySiu** [comment]
Just a heads up: I removed a lot of unneeded scss files and changed how we customize Materialize. https://github.com/earlymarket/CoPosition/commit/1d0827b4e97a2b7fb4374d06f53e8bf74dc5f2f3

It Shouldn't break anything that isn't already broken and it means we don't need to rely on `!important` rules any more. I hope this makes your job easier going forward.

**Comment by limzykenneth** [comment]
Great thanks. I'll have a look in a bit.

**Pull request merged**